tests/fixtures/v2/legacyCustomTiming.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Raygun4JS with V2 API</title>
+    <script src="/fixtures/common/instrumentXHRs.js"></script>
+    <script type="text/javascript">
+      !function(a,b,c,d,e,f,g,h){a.RaygunObject=e,a[e]=a[e]||function(){
+      (a[e].o=a[e].o||[]).push(arguments)},f=b.createElement(c),g=b.getElementsByTagName(c)[0],
+      f.async=1,f.src=d,g.parentNode.insertBefore(f,g),h=a.onerror,a.onerror=function(b,c,d,f,g){
+      h&&h(b,c,d,f,g),g||(g=new Error(b)),a[e].q=a[e].q||[],a[e].q.push({
+      e:g})}}(window,document,"script","/dist/raygun.min.js","rg4js");
+  </script>
+  </head>
+  <body>
+    <script type="text/javascript">
+      rg4js('apiKey', 'abcdef==');
+      rg4js('enableCrashReporting', false);
+      rg4js('enablePulse', true);
+      rg4js('options', {
+        pulseCustomLoadTimeEnabled: true
```

Comment:
I really don't like what this is called. It should be `pulseEnableCustomTimings`

Reply:
Agreed, this API however is for the legacy/existing custom timings